src/libgrate/grate-3d.c

```diff
@@ -659,11 +659,13 @@ static void grate_3d_set_texture_desc(struct host1x_pushbuf *pb,
 				      struct host1x_pixelbuffer *pixbuf,
 				      unsigned max_lod,
 				      unsigned wrap_mode,
-				      bool mip_filter,
-				      bool mag_filter,
-				      bool min_filter)
+				      bool mipmap_enabled,
+				      bool min_filter_linear_within,
+				      bool min_filter_linear_between,
+				      bool mag_filter_linear)
```

Comment:
This isn't really important, but I'm having a bit of difficulty liking this name-change. I know that I merged it in envytools, but I'm having a bit of second thought here... What's wrong with `mag`, `min`, `mip` names? These are the names that D3D uses for the filtering modes, I've never seen OpenGL refer to these apart from that wiki-site you listed (which is more of a textual explanation than "terminology")...

Basically, every time I read these, I get a little bit confused. I guess I'm just so used to the old names...